### DISCUSSION TITLE
ios端无法使用

### DISCUSSION THREAD

**Discussion by IT-yao** [1715173329/UnblockNeteaseMusic#711] [Q&A] [answered]
<img width="281" alt="微信图片_20220520201949" src="https://user-images.githubusercontent.com/82700864/169526778-479b8049-2007-4daa-9b3a-62f836da1e2e.png">

<img width="1044" alt="微信图片_20220520201731" src="https://user-images.githubusercontent.com/82700864/169526547-013c1adf-54ba-4f32-85d9-c5136c5091f0.png">

在window端可以使用，看了感觉图二歌连接好像有问题

**Comment by 1715173329** [comment, marked as answer]
#682 设置 endpoint

**Comment by IT-yao** [comment]
哥 ，endpoint这个docker 运行的 咋设置

**Comment by IT-yao** [comment]
> #682 设置 endpoint
可以了

**Reply by vleyong** [reply]
请问iOS的怎么部署呀，求文字简述一下，谢谢
